ubuntu/standard/4.0/Dockerfile

```
@@ -217,7 +217,7 @@ RUN set -ex \
 
 #****************      NODEJS     ****************************************************
 
-ENV NODE_12_VERSION="12.19.1" \
+ENV NODE_12_VERSION="12.21.0" \
```

```suggestion
ENV NODE_12_VERSION="12.22.2" \
```

Blog post on July 2021 Security Releases: https://nodejs.org/en/blog/vulnerability/july-2021-security-releases/